src/drivers/imu/adis16477/ADIS16477.cpp

```diff
@@ -358,16 +359,16 @@ ADIS16477::ioctl(struct file *filp, int cmd, unsigned long arg)
 					bool want_start = (_call_interval == 0);
 
 					/* convert hz to hrt interval via microseconds */
-					unsigned ticks = 1000000 / arg;
+					unsigned interval = 1000000 / arg;
 
 					/* check against maximum sane rate */
-					if (ticks < 1000) {
+					if (interval < 1000) {
```

Comment:
The ADIS16477 max conversion rate is 2000 sps.  This might be a good time to #define that value in this driver to get rid of a magic number here. (https://www.analog.com/media/en/technical-documentation/data-sheets/ADIS16477.pdf)

Reply:
Followup issue?